<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1422</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>871</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="buildingLabel">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>249</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose a building:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>40</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="houseSelectionButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>60</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Build a basic house. Cost: 50 Wood, 50 Stone&lt;/p&gt;&lt;p&gt;Houses conquer land in a 2 tile radius.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>Basic House</string>
    </property>
   </widget>
   <widget class="QToolButton" name="farmSelectionButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>150</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Build a Farm.&lt;/p&gt;&lt;p&gt;Cost: 50 Money, 50 Wood.&lt;/p&gt;&lt;p&gt;Farms can be as a one time source of 100 food. Does not conquer any land.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>Farm</string>
    </property>
   </widget>
   <widget class="QLabel" name="coordinateLabel">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(84, 82, 82);</string>
    </property>
    <property name="text">
     <string>coord</string>
    </property>
   </widget>
   <widget class="QToolButton" name="deleteBuildingButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>240</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete selected building. Does not remove conquered land, but does not refund any spent resources.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Delete Building</string>
    </property>
   </widget>
   <widget class="QToolButton" name="hqSelectionButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>60</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Build a Headquarters.&lt;/p&gt;&lt;p&gt;Cost: 500 Money, 100 Food, 500 Wood, 500 Stone&lt;/p&gt;&lt;p&gt;Headquarters allow you to build in the nearby area, and conquers land in a 3 tile radius.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>HQ</string>
    </property>
   </widget>
   <widget class="QToolButton" name="marketSelectionButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>150</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Build a Market&lt;/p&gt;&lt;p&gt;Cost: 50 Food, 50 Wood, 50 Stone&lt;/p&gt;&lt;p&gt;Market conquers land in a 2 tile radius and allows you to use the sell button.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>Market</string>
    </property>
   </widget>
   <widget class="QToolButton" name="bridgeSelectionButton">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>240</y>
      <width>121</width>
      <height>91</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Build a Bridge.&lt;/p&gt;&lt;p&gt;Cost: 20 Wood.&lt;/p&gt;&lt;p&gt;Bridges can be used to pass water areas. They also conquer land in a 1 tile radius.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>Bridge</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>118</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wood</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stone</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Food</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Money</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>70</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>100</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>155</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>50</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>50</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>50</y>
      <width>20</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="woodLabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="stoneLabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="foodLabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>110</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="moneyLabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoboxlabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>750</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="sellButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>50</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Sell 50 Wood and 50 Stone for 50 Gold. Requires the Market.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>Sell</string>
    </property>
   </widget>
   <widget class="QToolButton" name="buyButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>100</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Buy 50 Wood and 50 Stone for 50 Gold. Requires the Market.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="toolTipDuration">
     <number>5000</number>
    </property>
    <property name="text">
     <string>Buy</string>
    </property>
   </widget>
   <zorder>buildingLabel</zorder>
   <zorder>line</zorder>
   <zorder>houseSelectionButton</zorder>
   <zorder>farmSelectionButton</zorder>
   <zorder>coordinateLabel</zorder>
   <zorder>deleteBuildingButton</zorder>
   <zorder>hqSelectionButton</zorder>
   <zorder>marketSelectionButton</zorder>
   <zorder>bridgeSelectionButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
   <zorder>line_5</zorder>
   <zorder>line_6</zorder>
   <zorder>line_7</zorder>
   <zorder>line_8</zorder>
   <zorder>line_9</zorder>
   <zorder>line_10</zorder>
   <zorder>woodLabel</zorder>
   <zorder>stoneLabel</zorder>
   <zorder>foodLabel</zorder>
   <zorder>moneyLabel</zorder>
   <zorder>graphicsView</zorder>
   <zorder>infoboxlabel</zorder>
   <zorder>sellButton</zorder>
   <zorder>buyButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1422</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
